core/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
 
   // utilities
   compile 'commons-cli:commons-cli:1.3.1'
+  compile 'args4j:args4j:2.33'
```

Comment:
Uh wait, why are we adding another dependency!!??? Can we please not have 2 cli parsing utilities...